ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py

```diff
@@ -10,26 +10,25 @@
 )
 
 
-# layer_norm
```

Comment:
Again, let's undo all unrelated changes here and only add a new test for `batch_norm`. Thanks!